static-site/src/components/splash/index.jsx

```diff
@@ -54,86 +53,144 @@ class Splash extends React.Component {
         </FlexCenter>
 
         <HugeSpacer/>
-        {this.context.user && <UserGroups/>}
 
-        <ScrollableAnchor id={'ncov'}>
-          <Styles.H1>SARS-CoV-2 (COVID-19)</Styles.H1>
-        </ScrollableAnchor>
-
-        <FlexCenter>
-          <Styles.CenteredFocusParagraph>
-            We are incorporating SARS-CoV-2 genomes as soon as they are shared and providing analyses and situation reports.
-            In addition we have developed a number of resources and tools, and are facilitating independent groups to run their own analyses.
-            Please see the <Link to="/sars-cov-2">SARS-CoV-2 resources page</Link> for more information.
-          </Styles.CenteredFocusParagraph>
-        </FlexCenter>
-        <Cards
-          squashed
-          cards={nCoVCards}
-        />
-
-        <HugeSpacer/>
-
-        <ScrollableAnchor id={'pathogens'}>
-          <Styles.H1>Explore pathogens</Styles.H1>
-        </ScrollableAnchor>
-
-        <FlexCenter>
-          <Styles.CenteredFocusParagraph>
-            Genomic analyses of specific pathogens kept up-to-date by the Nextstrain team
-          </Styles.CenteredFocusParagraph>
-        </FlexCenter>
+        <div className="row">
+          {/* SARSCOV2 */}
+          <div className="col-md-6">
+            <ScrollableAnchor id={'ncov'}>
```

Comment:
We don't need scrollable anchors now that we have individual pages for each, but as they're already here there's no problem to keep them